src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -65,6 +65,7 @@ public MethodKind MethodKind
             public bool IsExtensionMethod
             {
                 get { return (_flags & IsExtensionMethodBit) != 0; }
+                set { _flags = value ? (_flags | IsExtensionMethodBit) : (_flags & ~IsExtensionMethodBit); }
```

Comment:
>set { _flags = value ? (_flags | IsExtensionMethodBit) : (_flags & ~IsExtensionMethodBit); } [](start = 16, length = 92)

Please remove the setter, this an immutable state of the symbol, it should not be mutated once accessed.